kie-drools-wb-parent/kie-drools-wb-home-page-product/src/main/resources/org/kie/workbench/drools/client/resources/i18n/Constants.properties

```diff
@@ -14,9 +14,11 @@
 # limitations under the License.
 #
 
-Heading=Welcome to KIE Drools Workbench
-SubHeading=KIE Drools Workbench offers a set of flexible tools, that support the way you need to work. Select a tool below to get started.
+Heading=Welcome to Red Hat jBoss BRMS
```

Comment:
Hi @tomasdavidorg , yes, this file is intended for the product version, as you can see by the name of the module that it is in (`kie-drools-wb-home-page-product`).